javalib/src/main/scala/java/util/Locale.scala

```diff
@@ -1,7 +1,165 @@
 package java.util
 
-final class Locale
+//import java.{util => ju}
```

Comment:
I left that line commented out as other commented methods return `ju.Set`. Other places in the code used that alias so I thought it was helpful.